### PR TITLE
Rich notifications sample

### PR DESCRIPTION
**Pull request by IanStanion-google**
Fixes: https://github.com/GoogleChromeLabs/Extension-Docs/issues/85

### CODE REVIEW

File: api-samples/richNotification/richNotifications.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# chrome.notifications
```
Comment (sebastianbenz) [inline api-samples/richNotification/richNotifications.md:1]: Please rename this file to readme.md

**Review by sebastianbenz — COMMENTED** [review]
I can't get this to work in OSX following the instructions, am I missing something?

**Comment by IanStanion-google** [comment]
@sebastianbenz Can you confirm your notifications are on for chrome in your Mac System Settings?

**Comment by sebastianbenz** [comment]
They are, got it working (didn't realize that my mac started muting all notifications as it's almost bedtime 🤦). Anyway, it seems like the sample uses APIs which are deprecated and most types are not supported on OSX. Please rework it to only show features that work cross platform. 

Please make sure to catch these kinds of things when updating the samples.

**Comment by sebastianbenz** [comment]
Discussed this with Oliver, another idea:

* don't demo deprecated features 
* make it more clear which platforms support which feature: have separate button for each notification type in a popup and label which platforms support which.

@IanStanion-google can you please update the sample accordingly. Thanks!

**Review by sebastianbenz — APPROVED** [review]
Nice!

**Pull request merged**